src/Microsoft.OpenApi.OData.Reader/Operation/EntityDeleteOperationHandler.cs

```diff
@@ -38,13 +38,13 @@ protected override void Initialize(ODataContext context, ODataPath path)
         /// <inheritdoc/>
         protected override void SetBasicInfo(OpenApiOperation operation)
         {
-            // Summary
-            operation.Summary = "Delete entity from " + EntitySet.Name;
-
             IEdmEntityType entityType = EntitySet.EntityType();
 
-            // Description
-            operation.Description = DeleteRestrictions?.Description; 
+            // Use Short Description as Summary
```

Comment:
[Nitpick] Let's try and be consistent with the comments across the `Summaries` and `Descriptions` sections